docs: restore to new project docs (#30525)
supabase/supabase: apps/docs/content/guides/platform/backups.mdx

After locking in the desired point in time to recover to, The Dashboard will then prompt for a review and confirmation before proceeding with the restoration. The project will be inaccessible following this. As such, do ensure to allot for downtime beforehand. This is dependent on the size of the database. The larger it is, the longer the downtime will be. Once the confirmation has been given, the latest physical backup available is downloaded to the project and the database is partially restored. WAL files generated after this physical backup up to the specified point-in-time are then downloaded. The underlying records of transactions in these files are replayed against the database to complete the restoration. The Dashboard will display a notification once the restoration completes.

## Restore to a new project

> Supabase provides a convenient way to restore data from an existing project into a completely new one. Whether you're using physical backups or Point-in-Time recovery (PiTR), this feature allows you to easily duplicate projects, perform safe testing, or recover data for analysis. This feature is exclusive to users on paid plans and requires that physical backups are enabled for the source project.

Author: Updated.

> Supabase provides a convenient way to restore data from an existing project into a completely new one. Whether you're using physical backups or Point-in-Time recovery (PITR), this feature allows you to duplicate projects with ease, perform safe testing, or recover data for analysis. This feature is exclusive to users on paid plans and requires that physical backups are enabled for the source project.

Reviewer: ```suggestion
Supabase provides a convenient way to restore data from an existing project into a completely new one. Whether you're using physical backups or Point-in-Time recovery (PITR), this feature allows you to duplicate project data with ease, perform testing safely, or recover data for analysis. This feature is exclusive to users on paid plans and requires that physical backups are enabled for the source project.
```

Author: Done.

<Admonition type="note">

> PiTR is an additional add-on available for organizations on a paid plan with physical backups enabled.

Author: Fixed.

</Admonition>

> To begin, switch to the source project—the project containing the data you wish to restore—and go to the [database backups](https://supabase.com/project/_/database/backups/restore-to-new-project) page. Select the **Restore to a New Project** tab.

Reviewer: ```suggestion
To begin, switch to the source project—the project containing the data you wish to restore—and go to the [database backups](https://supabase.com/dashboard/project/_/database/backups/restore-to-new-project) page. Select the **Restore to a New Project** tab.
```

Reviewer: @Crispy1975 i don't see this tab in prod Dashboard yet. are we waiting to release it?

Author: I'll double check the settings to be sure all is as it sold be.

Author: Done.

> To begin, navigate to the source project—the project containing the data you wish to restore—and head to the [database backups](https://supabase.com/project/_/database/backups/restore-to-new-project) page. Here, you’ll find the **Restore to a New Project** tab, which serves as the starting point for the process.

Reviewer 2: ```suggestion
To begin, switch to the source project (the project containing the data you wish to restore) and go to the [database backups](https://supabase.com/project/_/database/backups/restore-to-new-project) page. Select the **Restore to a New Project** tab.
```

Author: Updated.

> If the source project has physical backups available, a list of these backups will be displayed. Each entry will include a "Restore" button, enabling you to select the specific backup you want to use for the restoration. For projects with PiTR enabled, a date and time selector will be available which allows you to specify the exact point in time from which you wish to restore data.

Reviewer 2: ```suggestion
A list of available backups is displayed. Select the backup you want to use and click the "Restore" button. For projects with PITR enabled, use the date and time selector to specify the exact point in time from which you wish to restore data.
```

Author: Updated.

> Once you’ve made your choice, Supabase takes care of the rest. A new project is automatically created, replicating key configurations from the original, including the compute instance size, disk attributes (type, size, IOPs, throughput), SSL enforcement settings, and network restrictions. The data will remain in the same region as the source project to ensure compliance with data residency requirements. The entire process is fully automated simplifying the restoration workflow.

Reviewer 2: ```suggestion
Once you’ve made your choice, Supabase takes care of the rest. A new project is automatically created, replicating key configurations from the original, including the compute instance size, disk attributes (type, size, IOPs, throughput), SSL enforcement settings, and network restrictions. The data will remain in the same region as the source project to ensure compliance with data residency requirements. The entire process is fully automated.
```

Author: Done.
